src/utils/d3Graph.js

```diff
@@ -42,7 +40,6 @@ module.exports = {
     // TODO This is dirty
```

Comment:
This is a little dirty :) and I wouldn't fault you for leaving it as-is because it gets the job done and the alternative is a bunch of work. I'm definitely in favor of [keeping DOM manipulation in React land](http://www.macwright.org/2016/10/11/d3-and-react.html) as much as possible, and in this case the big gotcha with that approach would be `d3.brush`, which gives you a lot of functionality for free. You'd have to change interactions or clone [the technique d3.brush does](https://github.com/d3/d3-brush/blob/master/src/brush.js) in a React-friendly way, which would be tricky.